### ISSUE TITLE
ArcDps issue after Jan 1st 2023 GW2Launcher update

### ISSUE THREAD

**Issue by wetwillyhip** [Healix/Gw2Launcher#250]
Hi Healix,

So I run up to 8 accounts for login rewards and before the update, Arc Dps would run no problem on all of the accounts even though I didn't need it to. After the update (build 7075), the 1st account I run everytime runs ArcDps no problem but all others have read/write issues and Arc pops an error window on all the other accounts saying Error 23 - being used by another process. Is there a way to remedy this issue? I don't care if it runs on all accounts again, as long as there aren't error popups or even better, I'd like it to run on only ONE account. I have localized execution set to virtual at binary level to save disk space.

If I have to enable FULL level localized execution as part of a fix, I really don't want to -- don't want my 60gb .dat file to be duped a bunch taking up a TON of space. Thank for your help! Really love your GW2Launcher you created, truly is the best!

**Comment by Healix** [comment]
Localized execution set to Binaries or Full use the same actual disk space. Full only looks higher if you check the size of the folder, but in actuality, the Gw2.dat files in those folders are all the same file and don't take up any additional space. If you want to confirm it, check your disk usage under Computer, then delete Gw2.dat to see that it doesn't change.

Note Binaries only applies to DX9, so unless you're using DX9, it's effectively the equivalent of having localized execution disabled. Full is required for DX11, since DLLs go into the main folder, rather than the bin64 folder.

If you want Arc/addons on only 1 account, remove Arc from the main GW2 folder, set localized execution to Full and choose to only include the 1 account you want it on. Next to enabling localized execution should be a link to open or resync it (only if it has already been configured) - click resync, then open. OK out of the settings, then launch the account to have its # folder created (you don't need to login, it just needs to open the launcher). In the folder that was opened, open the # folder that was created for the account and treat this folder as the main GW2 folder for the account (place Arc here).

**Comment by wetwillyhip** [comment]
SUPER fast response and incredibly helpful! Worked like a charm Thank you thank you!

**Comment by bluenote73** [comment]
I have this same problem, but I just want the error to go away. I actually don't see localized execution mentioned anywhere so I don't know if I have it turned on or not? thanks

**Comment by Healix** [comment]
You can find it under Settings > Guild Wars 2 > Management

**Comment by bluenote73** [comment]
> You can find it under Settings > Guild Wars 2 > Management

Thank you. :) Sorry to bug you further. When arc is updated, or I need to manually disable addon manager since it doesn't disable properly, do I need to go into each acct config and resync? I seem to sometimes get this to work and then other times don't.
thx

**Comment by bluenote73** [comment]
I don't have syncing set to all but it does seem like I have to resync to for example today, get disabling arc to take effect.

There's a lot of options, do you have an opinion on a best practice setup when one is using arc?
thanks

**Comment by Healix** [comment]
It depends on what you want. All addons on all accounts, some addons on some accounts, just arc?

If you set sync to all and choose to delete files, all accounts would have the same addons and disabling arc would apply to all accounts. You would want your main account to be excluded from localized execution so it could serve as the master copy of the addons. This would be the simplest if you wanted all addons on all accounts.

If you want addons on only 1 account, set localized execution to only include that 1 account, then all addons go into that account's folder instead of the main GW2 folder (remove any addons from the main folder). You would set the addon manager to use this folder instead of the main one.

If you want addons on some but not all accounts, there isn't a simple way to do that. If all of the addons support sharing, you can use localized execution to only include accounts you don't want addons on, then all other accounts would share the addons in the main folder.

**Comment by Spughety** [comment]
> If you want Arc/addons on only 1 account, remove Arc from the main GW2 folder, set localized execution to Full and choose to only include the 1 account you want it on

Does it work if I do the inverse of this? Say I want Arc installed in the main GW2 folder, but all my alts have their own localized versions where it is not included.

**Comment by Healix** [comment]
> Does it work if I do the inverse of this? Say I want Arc installed in the main GW2 folder, but all my alts have their own localized versions where it is not included.

Yes. Set localized execution to exclude addons and select to exclude the main account (or only include the alts).

**Comment by bluenote73** [comment]
Thanks, I think I have my main set up as not localized and then everything else localized and that seems OK.
One problem I'm having that I don't know is related is that the "I am Evon Gnashblade" thing *never* persists past restart, and same with the Environmental Settings new slider. Can I fix this somehow? thx

**Comment by Healix** [comment]
GW2 won't save any of its local settings when multiple clients are allowed to launch. If you want those to save, right click the account > Selected > Launch (normal), which limits you to 1 account at a time.
